src/main/kotlin/org/radarbase/gateway/filter/KafkaTopicsAuthFilter.kt

```diff
@@ -0,0 +1,20 @@
+package org.radarbase.gateway.filter
+
+import jakarta.ws.rs.container.DynamicFeature
+import jakarta.ws.rs.container.ResourceInfo
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.FeatureContext
+import org.radarbase.gateway.resource.KafkaTopics
+import org.radarbase.jersey.auth.filter.AuthenticationFilter
+
+class KafkaTopicsAuthFilter(
+    @Context private val authFilter: AuthenticationFilter,
```

Comment:
@blootsvoets The `authFilter` here seems to not get resolved causing a NullPointerException. I couldn't figure out why.

Reply:
Indeed, I meant to return to this issue. Fixed in ec76b12.

Reply:
Thank you!